components/builder-originsrv/src/migrations/2018-03-05-224023_origin_secrets/down.sql

```diff
@@ -0,0 +1 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Are we leaving these empty?  If so, would be good to add that prescriptive guidance to the migrations doc if we are not able to take advantage of the diesel rollbacks.

Reply:
Yeah the rollbacks are sort of pointless for our usecase. I'll add a card to write up a doc on it

Reply:
https://github.com/habitat-sh/builder/issues/275